LibOS/shim/src/bookkeep/shim_signal.c

```diff
@@ -246,6 +246,15 @@ static void divzero_upcall (PAL_PTR event, PAL_NUM arg, PAL_CONTEXT * context)
 
 static void memfault_upcall (PAL_PTR event, PAL_NUM arg, PAL_CONTEXT * context)
```

Comment:
Can you add some comments about the role of this function, when it is likely called, and how the arguments should be interpreted?

Reply:
Already added.